microservices/google-api/google_search.json

```diff
@@ -4,6 +4,9 @@
     "sites":[
         { 
         "name":"https://www2.gov.bc.ca/"
+        },
+        { 
+        "name":"https://efficiencybc.ca/"
```

Comment:
It looks as though there is nothing more recent than 2019-03-02. Per configuration: https://github.com/bcgov/GDX-Analytics/tree/master/microservices/google-api#configuration we can do that with the following on line 10, on a line after `"name":"https://efficiencybc.ca/",`
```
"start_date_default":"2019-03-02"
```
@danpollock what do you think?

Reply:
You mean older than 2019-03-02? 

Yeah. That's a good change.

Reply:
Yes, that is what I meant. Looks good.